pkg/keycloak/keycloak.go

```diff
@@ -410,24 +409,56 @@ func (h *Handler) reconcileClients(kc *v1alpha1.Keycloak, kcClient KeycloakInter
 
 	clientPairsList := map[string]*v1alpha1.KeycloakClientPair{}
 	for i := range objRealm.Clients {
-		client := objRealm.Clients[i]
+		client := &objRealm.Clients[i]
 		clientPairsList[client.ClientID] = &v1alpha1.KeycloakClientPair{
-			ObjClient: &client,
-			KcClient:  kcClients[client.ClientID],
+			SpecClient: client,
+			KcClient:   kcClients[client.ClientID],
 		}
 		delete(kcClients, client.ClientID)
 	}
 
 	for i := range kcClients {
 		client := kcClients[i]
 		clientPairsList[client.ClientID] = &v1alpha1.KeycloakClientPair{
-			KcClient:  client,
-			ObjClient: nil,
+			KcClient:   client,
+			SpecClient: nil,
```

Comment:
You need to put this back to be ObjClient, think you might have copied this form my PR where there were additional changes to the struct.

Reply:
I changed the struct as well: https://github.com/aerogear/keycloak-operator/pull/21/commits/bf8e581560ca46d133c7b448bae5d8f7f7b71bbc#diff-8edf0a22709a32db81dce9a863dcca19R90